monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -515,8 +449,6 @@ def generate_report():
             "glance": {
                 "scanned": scanned_nodes,
                 "exploited_cnt": exploited_cnt,
-                "stolen_creds": stolen_creds,
-                "ssh_keys": extract_ssh_keys(stolen_creds),
                 "strong_users": PTHReportService.get_strong_users_on_crit_details(),
```

Comment:
What's "crit"? Critical? Maybe rename this while we're at it.

Reply:
I talked with Vakaris and maybe that `PTHReportService` is just not working